monai/visualize/img2tensorboard.py

```diff
@@ -9,7 +9,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional, Sequence, Union
+from typing import Optional, Sequence, Union, TYPE_CHECKING
```

Comment:
I think we can remove line 182, in that case is it possible to remove line 26?

Reply:
> I think we can remove line 182, in that case is it possible to remove line 26?

Yeah 182 is the only time it is used in a runtime context. I didn't know the importance of the assert so I didn't touch it.

Reply:
Removed it